conf/pxe_cluster/grub.cfg

```diff
@@ -0,0 +1,5 @@
+menuentry "Install Ubuntu" {
+set gfxpayload=keep
+linux ubuntu-installer/amd64/linux gfxpayload=800x600x16,800x600 live-installer/net-image=http://172.16.141.17/ubuntu/install/filesystem.squashfs console=ttyS1,115200 console=ttyS0,115200 console=tty ramdisk_size=16432 root=/dev/rd/0 rw --- auto=true url=http://172.16.141.17/ubuntu/preseed/hwe-ubuntu-server.seed quiet
```

Comment:
Do we need quiet enabled?  Won't this just be used for installations and having more info may be helpful when debugging.

Reply:
A license header would not hurt.  Please add it.